ideascube/blog/forms.py

```diff
@@ -15,6 +15,7 @@ class Meta:
             # control over the format to bypass L10N.
             "published_at": forms.DateInput(format='%Y-%m-%d'),
             "lang": LangSelect,
+            "text": RichTextEntry(with_media=True)
```

Comment:
I don't understand how to type text in the blog now.

There's no toolbar to make the text bold or italic, no way to insert hyperlinks.

And I only get the button to add medias as a floating button next to the text cursor if the content is empty.

Is that really expected? 😮